docs/repo/property-pages/README.md

```diff
@@ -12,6 +12,12 @@ This documentation details the updated Project Properties UI and associated back
 4. Extensibility. We need to support 3rd parties (both internal and external) adding both entirely new pages _and_ customizing the pages we provide. 
 5. Support new functionality. We want to enable scenarios such as searching for properties, editing the MSBuild expressions that underpin many properties, and collecting the most commonly used properties in one area.
 
+## How-To Guides
+
+These guides describe how to get specific scenarios up and running, step by step.
+
+- [HOW TO: Add a new property page](how-to-add-a-new-property-page.md)
+
```

Comment:
I'm not sure we need a dedicated 'how to' section. It splits our documentation for extenders/customisation in two.

I'd vote for just moving the _Customising Project Properties_ section above the _Architecture_ section and adding the new how-to document as a bullet point to that list.